Dream News/dist/metadata.json

```diff
@@ -9,20 +9,20 @@
     }],
     "service": "Dream News",
     "description": {
-        "en": "Top news for anime, mangá, light novel and everything related to this world! Read on your Computer, Notebook, Smartphone or any other Mobile Device.",
+        "en": "Main news about animes, mangas, light novels and everything related to the Otaku world! Read on your Computer, Notebook, Smartphone or any other Mobile Device.",
         "pt_BR": "Principais notícias sobre animes, mangás, light novel e tudo relacionado ao mundo Otaku! Leia em seu Computador, Notebook, Smartphone ou qualquer outro Dispositivo Móvel."
     },
     "url": [
         "www.dreamnews.live"
     ],
-    "version": "1.1",
-    "logo": "https://menma.is-inside.me/n1DGYByn.png",
+    "version": "1.1.1",
+    "logo": "https://menma.is-inside.me/lcMhkI1e.png",
     "thumbnail": "https://menma.is-inside.me/M6AvnGvt.png",
     "color": "#0c8a2e",
     "tags": [
         "anime",
         "manga",
         "news"
     ],
-    "category": "anime"
+    "category": "other"
```

Comment:
it is still related to anime isnt it? so why change it.

Reply:
not 100%, i asked one of their staff team and said that the website was not only about anime, they talk about video games and other stuff as well, if you check the website.

Reply:
Yea I checked the website and all I could see were  anime pictures xD